nextcord/__main__.py

```diff
@@ -155,7 +156,7 @@ async def cog_after_invoke(self, ctx):
 # certain file names and directory names are forbidden
 # see: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247%28v=vs.85%29.aspx
 # although some of this doesn't apply to Linux, we might as well be consistent
-_base_table = {
+_base_table: dict[str, str | None] = {
```

Comment:
Wasn't this Union syntax introduced in Python 3.10?

Reply:
Correct, note the future import that has been supported since 3.7 https://peps.python.org/pep-0563/#enabling-the-future-behavior-in-python-3-7

Reply:
Oh right. I forgot about that.